dashboard/web/store/wallet.js

```diff
@@ -39,6 +40,7 @@ const actions = {
   async createWallet({commit, rootState}, {password, beneficiaryAddr}) {
     try {
       let res = await this.$axios.post('/api/wallet/create', {password: password, beneficiaryAddr:beneficiaryAddr})
+      localStorage.setItem('seed', seedHash(password))
```

Comment:
Are we storing password in local storage? This seems to be very dangerous as anyone who has access to the computer will now be able to see the password plaintext.